include/revng/Support/GzipStream.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/Support/raw_ostream.h"
+
+void gzipCompress(llvm::raw_ostream &OS, const llvm::ArrayRef<uint8_t> Buffer);
```

Comment:
Doc and rename arguments to explicitly reflect who's the input and who's the output.

Reply:
Done